packages/api/src/schema/api-schema.yaml

```diff
@@ -0,0 +1,3102 @@
+                type: array
+                items:
+                  $ref: "#/components/schemas/object-store"
+        default:
+          description: Error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/error"
+  /multistream/target:
+    get:
+      description: Lists existing multistream targets
+      responses:
+        "200":
+          description: Success
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/multistream-target"
+        default:
+          description: Error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/error"
+    post:
+      description: Creates new multistream target
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/multistream-target"
+      responses:
+        "201":
+          description: Success
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/multistream-target"
+        default:
+          description: Error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/error"
+  "/multistream/target/{id}":
+    parameters:
+      - name: id
+        description: ID of the multistream target
+        in: path
+        required: true
+        schema:
+          type: string
+    get:
+      description: Gets a specific multistream target details
+      responses:
+        "200":
+          description: Success
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/multistream-target"
+        default:
+          description: Error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/error"
+    patch:
+      description: Enables or disables an existing multistream target
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/multistream-target-patch-payload"
+      responses:
+        "204":
+          description: Success
+        default:
+          description: Error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/error"
+    delete:
+      description: Deletes an existing multistream target
+      responses:
+        "204":
+          description: Success
+        default:
+          description: Error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/error"
+  /user-verification:
```

Comment:
This is also an internal API, called only by our own dashboard. We don't really want this in our API reference.